### PR TITLE
Use hash.Hash type in service proto definition

### PR DESCRIPTION
**Pull request by krhubert**

### CODE REVIEW

File: hash/hash.go [hunk 1]
```diff
@@ -95,6 +95,11 @@ func (h Hash) Marshal() ([]byte, error) {
 	return h, nil
 }
 
+// MarshalTO marshals hash into slice of bytes. It's used by protobuf.
```
Comment (NicolasMahe) [inline hash/hash.go]:
```suggestion
// MarshalTo marshals hash into slice of bytes. It's used by protobuf.
```

**Comment by NicolasMahe** [comment]
@krhubert please provide description and metadata even if this PR will be merge into another one. I didn't see this a first glance.

**Comment by krhubert** [comment]
> @krhubert please provide description and metadata even if this PR will be merge into another one. I didn't see this a first glance.

The full description is in the title. I don't know what more needs to be wrote here. And I don't understand about what metadata are you talking about?

**Comment by antho1404** [comment]
the labels/project/milestone etc...

**Comment by krhubert** [comment]
> the labels/project/milestone etc...

I thought that pr that are merge to non-dev branch dosen't have to have all such stuff.

> Also, you can add in the description of how we can test that

Do we really want to waste time to write such obvious thing how to test change of API? Everyone knows how to test it...

**Pull request merged**